docs/windows_developer_guide.md

```diff
@@ -61,14 +61,6 @@ python --version
 pip install numpy==1.23
 ```
 
-## MSYS
-Download and install in c:\opt\msys64 https://github.com/msys2/msys2-installer/releases/download/2024-07-27/msys2-x86_64-20240727.exe
```

Comment:
Not sure if this part should be permanently deleted.

Reply:
Changed order in install_deps so that msys is available at the start.